runner/src/dml-handler/dml-handler-fixture.ts

```diff
@@ -0,0 +1,360 @@
+import { AST, Parser } from "node-sql-parser";
+import { TableDefinitionNames } from "../indexer";
+import { PostgresRow, WhereClauseMulti, WhereClauseSingle } from "./dml-handler";
+import { DmlHandlerI } from "./dml-handler";
+
+// TODO: Define class to represent specification
+interface TableSpecification {
+  tableName: string
+  columnNames: string[]
+  primaryKeyColumns: string[]
+  serialColumns: string[]
+}
+
+class PostgresRowEntity {
+  data: PostgresRow;
+  private primaryKeys: string[];
+
+  constructor(data: any, primaryKeys: string[]) {
```

Comment:
```suggestion
  constructor(data: Record<string, any>, primaryKeys: string[]) {
```
Would this be more correct?

Reply:
Oops! You're correct. I even defined an exlicit type to represent it but I must have forgotten to put it here during the refactoring!